modules/recovery/CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+# Version 0.1.0
```

Comment:
In the contract, they use `0.0.1`

https://github.com/5afe/CandideWalletContracts/blob/113d3c0/contracts/modules/social_recovery/SocialRecoveryModule.sol#L12

Reply:
I would keep our versioning and their versioning separate.